framework/src/media/utils/MediaUtils.cpp

```diff
@@ -36,6 +41,26 @@ static const std::string MPEG_MIME_TYPE = "audio/mpeg";
 static const std::string MP4_MIME_TYPE = "audio/mp4";
 static const std::string OPUS_MIME_TYPE = "audio/opus";
 
+unsigned int CRC32_MPEG2(unsigned char *data, unsigned int length)
```

Comment:
we have a crc32 in lib_crc32.c already.

Reply:
crc32 for mpeg2 is different with crc32() in lib_crc32.c.
I tried it, but verify failed.